source/blog/2017-03-13-ember-2-12-released.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Ember 2.12 and 2.13 Beta Released
+author: Godfrey Chan, Brendan McLoughlin
+tags: Releases
+---
+
+Today, the Ember project is releasing Ember.js, Ember Data and Ember CLI
+version 2.12.0.
+
+This also kicks off the 2.13 beta cycle for all sub-projects. We encourage our
+community (especially addon authors) to help test these beta builds and report
+any bugs before they are published as a final release in six weeks' time. The
+[ember-try](https://github.com/ember-cli/ember-try) addon is a great way to
+continuously test your projects against the latest Ember releases.
+
+You can read more about our general release process here:
+
+- [Release Dashboard](http://emberjs.com/builds/)
+- [The Ember Release Cycle](http://emberjs.com/blog/2013/09/06/new-ember-release-process.html)
+- [The Ember Project](http://emberjs.com/blog/2015/06/16/ember-project-at-2-0.html)
+- [Ember LTS Releases](http://emberjs.com/blog/2016/02/25/announcing-embers-first-lts.html)
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 2.12
+
+The
+2.12.0 release is an Ember.js Long-term support candidate. In six weeks, the 2.12.x series
+will become the latest LTS release and six weeks after that the 2.8 LTS branch
+will no longer receive bugfix patches.
+
+For more information about Ember's LTS policies, see the
+[announcement blog
+post](http://emberjs.com/blog/2016/02/25/announcing-embers-first-lts.html) and
+[builds page](http://emberjs.com/builds/).
+
+Ember 2.12 implements the `factoryFor` API as described in [RFC #150](https://github.com/emberjs/rfcs/blob/master/text/0150-factory-for.md).
+This public API replaces the intimate API of `_lookupFactory`, and additionally
+discourages developers from setting properies on classes returned from the
+container. For more information about this API see the [API
+docs](http://emberjs.com/api/classes/ContainerProxyMixin.html#method_factoryFor)
```

Comment:
This documentation URL has no content, needs to be written per TODOs in the issue description.